spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -138,33 +138,44 @@
     end
   end
 
-  context 'when _noGenerateTstatsHomePath is set to true for specific indexes' do
+  context 'when _noGenerateTstatsHomePath is set to true and _is_s2Index is set to true for specific indexes' do
```

Comment:
rather than conflating these contexts in the same tests, I would prefer to have separate contexts to test the new settings.

Reply:
I have created separate test now for [rspec_tests](https://github.com/cerner/cerner_splunk/pull/241/commits/eb38d4732d72ac6a0ce03a1c28f55e5591cc2868#diff-2dbf75fe595324af1923d6c51037b9e78cab1adcc9d842af08ba7e0efba026d4R226-R306)